test/common/index.js

```diff
@@ -70,7 +70,7 @@ function parseTestFlags(filename = process.argv[1]) {
   fs.closeSync(fd);
   const source = buffer.toString('utf8', 0, bytesRead);
 
-  const flagStart = source.indexOf('// Flags: --') + 10;
+  const flagStart = source.search(/\/\/ Flags:\s+--/) + 10;
```

Comment:
there was a small difference between how `Flags` are handled in python vs in `common/index.js` - this fixes that